src/main/java/frc/robot/subsystems/swerve/module/Module.java

```diff
@@ -114,8 +118,8 @@ public void updateInputs() {
 		moduleInputs.targetState = targetState;
 
 		Logger.processInputs(constants.logPath(), moduleInputs);
-		Logger.processInputs(driveStuff.logPath(), driveInputs);
-		Logger.processInputs(driveStuff.logPath(), couplingInputs);
+		Logger.processInputs(constants.logPath() + "/Drive", driveInputs);
+		Logger.processInputs(constants.logPath() + "/Drive", couplingInputs);
```

Comment:
```suggestion
		Logger.processInputs(constants.driveLogPath(), driveInputs);
		Logger.processInputs(constants.driveLogPath(), couplingInputs);
```
add in ModuleConstants